karaf/init/pom.xml

```diff
@@ -15,11 +15,6 @@
     <dependencies>
         <dependency>
             <groupId>io.cloudsoft.brooklyn.tosca</groupId>
-            <artifactId>brooklyn-tosca-karaf-deps-brooklyn</artifactId>
```

Comment:
It's as you noted below, really all that matters is that Karaf is able to satisfy the Import-... declarations in the brooklyn-tosca-karaf-init manifest.